### ISSUE TITLE
No default export, problem with export locals - can't use full classNames

### ISSUE THREAD

**Issue by SatarisGIT** [Jimdo/typings-for-css-modules-loader#87]
Hello!

I have problem with newest version of this loader.

Loader working great, but dont generate "export default locals;" string.

```css
export interface IMonitoringHeaderModuleScss {
  'header': string;
  'header__header_element': string;
  'headerHeaderElement': string;
  'header__header_element--modifier': string;
  'headerHeaderElementModifier': string;
}
```

export const locals: IMonitoringHeaderModuleScss;`

So I cant use any styles from it beacuse my styles when I import them

`import styles from './MonitoringHeader.module.scss';`

lands DIRECTLY in "styles" object, not "styles.locals"...

I dont like namedExport with camelCase - generated names look really bad with BEM convention.

**Comment by ztolley** [comment]
Same here +1

**Comment by dmnn** [comment]
+1

**Comment by nickpape** [comment]
Yep, I am having this issue too. I will try downgrading and seeing if that resolves issue.
